include/zephyr/drivers/misc/ft8xx/ft8xx_reference_api.h

```diff
@@ -171,6 +171,144 @@ static inline void cmd_swap(void)
+ * @ref FT8XX_OPT_FLAT removes the 3D effect and its value is 256.
+ *
+ * In @p s, a character value of 255 (it can be written as @c \\xff ) separates
+ * the off and on labels.
+ *
+ * @param x x-coordinate of top-left of toggle, in pixels
+ * @param y y-coordinate of top-left of toggle, in pixels
+ * @param w Width of toggle, in pixels
+ * @param font Font to use for text, 0-31. 16-31 are ROM fonts
+ * @param options Options to apply
+ * @param state State of the toggle: 0 is off, 65535 is on
+ * @param s String label for toggle
+ */
+static inline void cmd_toggle(int16_t x,
+			      int16_t y,
+			      int16_t w,
+			      int16_t font,
+			      uint16_t options,
+			      uint16_t state,
+			      const char *s)
+{
+	ft8xx_copro_cmd_toggle(x, y, w, font, options, state, s);
+}
+
+/**
+ * @brief Track touches for a graphics object
+ *
+ * This command will enable co-processor engine to track the touch on the
+ * particular graphics object with one valid tag value assigned. Then,
+ * co-processor engine will update the REG_TRACKER periodically with the frame
+ * rate of LCD display panel.
+ *
+ * Co-processor engine tracks the graphics object in rotary tracker mode and
+ * linear tracker mode:
+ *
+ * - Rotary tracker mode
+ *   Track the angle between the touching point and the center of graphics
+ *   object specified by @p tag value. The value is in units of 1/65536 of a
+ *   circle. 0 means that the angle is straight down, 0x4000 left, 0x8000 up,
+ *   and 0xC000 right from the center.
+ * - Linear tracker mode
+ *   If @p w is greater than @p h, track the relative distance of touching point
+ *   to the width of graphics object specified by @p tag value. If @p w is not
+ *   greater than @p h, Track the relative distance of touching point to the
+ *   height of graphics object specified by @p tag value. The value is in units
+ *   of 1/65536 of the width or height of graphics object. The distance of
+ *   touching point refers to the distance from the top left pixel of graphics
+ *   object to the coordinate of touching point.
+ *
+ * For linear tracker functionality, @p x represents x-coordinate of track area
+ * top-left and @p y represents y-coordinate of track area top-left. Both
+ * parameters are in pixels.
+ *
+ * For rotary tracker functionality, @p x represents x-coordinate of track area
+ * center and @p y represents y-coordinate of track area center. Both
+ * parameters are in pixels.
+ *
+ * @note A @p w and @p h of (1,1) means that the tracker is rotary, and reports
+ *       an angle value in REG_TRACKER. A @p w and @p h of (0,0) disables the
+ *       track functionality of co-processor engine.
+ *
+ * @param x x-coordinate
+ * @param y y-coordinate
+ * @param w Width of track area, in pixels.
+ * @param h Height of track area, in pixels.
+ * @param tag Tag of the graphics object to be tracked, 1-255
+ */
+static inline void cmd_track(int16_t x,
+			     int16_t y,
+			     int16_t w,
+			     int16_t h,
```

Comment:
single letter arguments are ok for local variables, bad idea for external functions, x and y are obviously ok "w" "h" "s" should really be expanded, I imagine you did it because that's how the existing apis are declared but I'd suggest changing them all to something meaningful, it's not like it would break existing code anyway